.github/workflows/release.yaml

```diff
@@ -1,36 +1,48 @@
 name: Release
 
 on:
-  push:
-    tags:
-      - '*.*.*'
+  release:
+    types: [published]
```

Comment:
So, would have to create the release manually after a tag is pushed?

Reply:
So release workflow would be

1. Create release PR (version bump, changelog updates).
2. Merge.
3. Create release via UI w/ changelog.

I am guessing later we can do a workflow similar to backport where we can label it as a release so it triggers a creation of a tag from the version and corresponding release.

Reply:
The release can create the tag; I am going to suggest that we restrict tags such that they can only be created from the web UI.